web/react/components/navbar_dropdown.jsx

```diff
@@ -154,11 +154,14 @@ export default class NavbarDropdown extends React.Component {
                 </li>
             );
 
-            this.state.teams.forEach((teamName) => {
-                if (teamName !== this.props.teamName) {
-                    teams.push(<li key={teamName}><a href={Utils.getWindowLocationOrigin() + '/' + teamName}>{'Switch to ' + teamName}</a></li>);
+            for (let teamId in this.state.teams) {
```

Comment:
Can we sort them by team display name so they always show in alphabetical order.